### ISSUE TITLE
Tensorflow NMT Tutorial - Why there is no Validation set Included?

### ISSUE THREAD

**Issue by haqkiemdaim** [tensorflow/text#986]
Hi there!

I just want to ask regarding this tutorial:
[https://github.com/tensorflow/text/blob/master/docs/tutorials/nmt_with_attention.ipynb]() 

I'm very new to NMT but why in this tutorial only used Training Set and No Validation Set included? Any reason for this?

I wonder how can we evaluate the model without having validation split.

**Comment by broken** [comment]
I believe this tutorial along with a few others are mainly concerned with showing how you would construct the model and train it. Likely if you were to add it to production, you would also have a set for performing evaluations.

ps. Those tutorials can also be found on tensorflow.org/text (https://www.tensorflow.org/text/tutorials/nmt_with_attention).
